services/common/src/components/forms/RenderDate.tsx

```diff
@@ -14,6 +14,7 @@ interface DateInputProps extends BaseInputProps {
   yearMode?: boolean;
   disabledDate?: (currentDate) => boolean;
   formatViewDate?: boolean;
+  fieldEditMode?: boolean;
```

Comment:
This is to allow specific fields within a form to individually set their editMode (ie. Nested in a step form)

Reply:
I think this might make sense to move into `BaseInputProps`

Reply:
Good call!